src/Util.cpp

```diff
@@ -529,7 +528,7 @@ struct TickStackEntry {
     int line;
 };
 
-vector<TickStackEntry> tick_stack;
```

Comment:
Throws a warning about missing extern and that it should be static if module-local, which it is. No need to export this.